src/main/resources/db/changelog/0011-add-key-constraint-user-and-financial-transaction-category.yaml

```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
+  - changeSet:
+      id: "add foreign key to financial_transaction_categories"
+      author: SzanieckiFranciszek
```

Comment:
Please delete "-" mark

Reply:
Hi, I removed "_" from line 3, this is what you means? In line 2, mark "-" is it good practise followed in other .yaml file.